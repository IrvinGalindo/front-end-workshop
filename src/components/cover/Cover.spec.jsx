import React from 'react';
import { shallow } from 'enzyme';

import Cover from './Cover';

describe('Cover', () => {
  const cover = shallow(<Cover />);

  it('should render', () => {
    expect(cover).toMatchSnapshot();
  });
});
