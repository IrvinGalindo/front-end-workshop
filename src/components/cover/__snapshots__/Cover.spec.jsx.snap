// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover should render 1`] = `
<div
  className="yta_Cover"
>
  <img
    alt="Hogares para nuestros amigos"
    className="yta_Cover-logo"
    src="logo.svg"
  />
  <video
    autoPlay={true}
    className="yta_Cover-video"
    loop={true}
    muted={true}
  >
    <source
      src="puppiness.ogv"
      type="video/ogg"
    />
    <source
      src="puppiness.webm"
      type="video/webm"
    />
    <source
      src="puppiness.mp4"
      type="video/mp4"
    />
  </video>
</div>
`;
